### DISCUSSION TITLE
Query without a variable

### DISCUSSION THREAD

**Discussion by gabriiels** [urql-graphql/urql#2954] [Q&A] [answered]
Hello!

How do you do a query without a variable using urql/core ?

For example, for this query:

```export const COFFEE = gql`
   query {
     coffees {
       id
       name
       brand
       createdAt
     }
   }
`;```

**Comment by JoviDeCroock** [comment, marked as answer]
You can call [`.query`](https://formidable.com/open-source/urql/docs/api/core/#clientquery) with an empty object

**Reply by gabriiels** [reply]
this way :

client.query(COFFEE, {}).toPromise(); ?
